include/swift/Serialization/ModuleFormat.h

```diff
@@ -614,6 +615,10 @@ namespace options_block {
     IS_TESTABLE
   >;
 
+  using ArePrivateImportsEnabledLayout = BCRecordLayout<
+    ARE_PRIVATE_IMPORTS_ENABLED
+  >;
```

Comment:
What do you think about using the presence of the other table to do this, and leaving it out of the options section altogether?